packages/web-frontend/src/containers/MapTableModal/MapTableModal.js

```diff
@@ -21,8 +21,8 @@ import {
 import { Tooltip } from '../../components/Tooltip';
 
 const IconButton = styled(MuiIconButton)`
-  margin: 0 0 0 1rem;
-  padding: 10px 8px 10px 12px;
+  margin: 0;
+  padding: 0;
```

Comment:
setting the padding to 0 makes the button really small which makes it harder to click and look a bit funny on hover. I think the best thing to do would be to use `MuiIconButton` directly for this button and add any padding tweaks to the container to get it in the right place.

Reply:
Yeah that's right! I haven't thought of that. Since we already have `MuiIconButton` here, I will just set the `margin-right` to meet UX design.